frontend/src/plugins/__tests__/preferences.spec.js

```diff
@@ -0,0 +1,448 @@
+import { getters, loadFromLocalStorage, mutations } from '@/plugins/store/preferences'
+
+let originalLocalStorage
+beforeEach(() => {
+  originalLocalStorage = window.localStorage
+  window.localStorage = (() => {
+    let store = {}
+
+    return {
+      getStore: () => store,
+      getItem: (key) => store[key] ?? null,
+      setItem: (key, value) => {
+        store[key] = value?.toString() ?? 'undefined'
+      },
+      removeItem: (key) => {
+        delete store[key]
+      },
+      clear: () => {
+        store = {}
+      },
+      key: () => '',
+      length: Object.keys(store).length,
+    }
+  })()
+})
+
+afterEach(() => {
+  window.localStorage = originalLocalStorage
+})
+
+describe('reading state', () => {
+  it('loads saved picasso edit mode true', async () => {
+    // given
+    const state = loadFromLocalStorage({
+      'preferences:/camps/1a2b3c4d:picassoEditMode': 'true',
```

Comment:
minor:
the prefix could be extracted into a constant to simplify changing it
the camp iri could also be a constant, that you see from afar if two different constants point to the same camp or not.

Reply:
I kind of like the explicit-ness of the test this way, and I think a search and replace would work well if we change the prefix. But it is a matter of taste. I will replace the camp URI with a constant, but since template literals can't directly be used as keys in JS objects it will look like this:
```js
const state = loadFromLocalStorage({
  [`preferences:${CAMP_URI}:picassoEditMode`]: 'true',
})
```

Reply:
as you like, it is a minor taste related issue